### PR TITLE
[CU-346b87e] Merge 'main' into 'centauri' branch

### PR DESCRIPTION
**Pull request by obsessed-cake**
## Issue
https://app.clickup.com/t/346b87e

## Checklist

- [ ] Updated the rust/typescript docs.
- [ ] Updated the `docs/`.

### CODE REVIEW

**Review by blasrodri — APPROVED** [review]
LGTM

**Review by dzmitry-lahoda — APPROVED** [review]

File: code/utils/subxt-exports/src/generated/composable_dali_dev.rs [hunk 1]
```diff
@@ -1541,10 +1541,10 @@ pub mod api {
             use super::runtime_types;
             pub struct ConstantsApi;
             impl ConstantsApi {
-                #[doc = " A fee mulitplier for `Operational` extrinsics to compute \"virtual tip\" to boost their"]
+                #[doc = " A fee multiplier for `Operational` extrinsics to compute \"virtual tip\" to boost their"]
                 #[doc = " `priority`"]
                 #[doc = ""]
-                #[doc = " This value is multipled by the `final_fee` to obtain a \"virtual tip\" that is later"]
+                #[doc = " This value is multiplied by the `final_fee` to obtain a \"virtual tip\" that is later"]
```
Comment (dzmitry-lahoda) [inline code/utils/subxt-exports/src/generated/composable_dali_dev.rs:1547]: likely `generated` must be excluded from cspell all togethere - like composable-subxt generated for example

**Review by dzmitry-lahoda — APPROVED** [review]

File: code/utils/xcmp/awesome-xcmp.md [hunk 2]
```diff
@@ -14,24 +14,31 @@ XCMP can be upward (parachain to relay), downward(relay to parachain) and siblin
 
 ## Conceptual
 
+<<<<<<< HEAD
 [Polkadot's Cross-chain Message Passing Protocol: Shared Security and Polkadot's Design](https://www.youtube.com/watch?v=XU6dAAQD9UE)
+=======
+[Polkadot's Cross-chain Message Passing Protocol: Shared Security and Polkadot's Design(video)](https://www.youtube.com/watch?v=XU6dAAQD9UE)
+>>>>>>> main
```
Comment (Martin-Kiss) [inline code/utils/xcmp/awesome-xcmp.md:21]:
```suggestion
[Polkadot's Cross-chain Message Passing Protocol: Shared Security and Polkadot's Design(video)](https://www.youtube.com/watch?v=XU6dAAQD9UE)
```
Artifact from merge conflict

File: code/utils/subxt-exports/src/generated/composable_dali_dev.rs (hunk 1 above)
Comment (obsessed-cake) [reply code/utils/subxt-exports/src/generated/composable_dali_dev.rs:1547]: done

File: code/integration-tests/local-integration-tests/README.md [hunk 3]
```diff
@@ -46,11 +14,18 @@ Add barrier and ED tests
...
\ No newline at end of file
+Make all tests using calculated fees our of types, not hardcoded (can be used in future to build RPC for fee calculator).
+
+In each reserver transfer test ensure sibling accounts, para accounts are set up well and total supplys always same in the end.
+
+Remove all usage of XTokens as these violate XCM guidlines and do not work well.
+
+Ensure that assets from pallets encoded like (parent = 0, pallet = 44, index = 1232132).
+
+Run attacks which try to spend parachain account, spend other person amount.
```
[the remark this answers is not in the document]
Comment (obsessed-cake) [reply code/integration-tests/local-integration-tests/README.md]: done.

File: code/utils/xcmp/awesome-xcmp.md (hunk 2 above)
Comment (obsessed-cake) [reply code/utils/xcmp/awesome-xcmp.md]: fixed.

**Pull request merged**